### ISSUE TITLE
Compress.getThumbnail error: type 'Future<String?>' is not a subtype of type 'FutureOr<String>' in type cast

### ISSUE THREAD

**Issue by MaxSchilling** [jonataslaw/VideoCompress#105]
After the upgrade to null safety and version 3.0.0 I now get an error when trying to get the thumbnail file from the video File. Compression on the main file seems to work - I just get an error on the getThumbnail method.

flutter: ----------------FIREBASE CRASHLYTICS----------------
flutter: type 'Future<String?>' is not a subtype of type 'FutureOr<String>' in type cast
flutter: 
#0      Compress.getFileThumbnail (package:video_compress/src/video_compress/video_compressor.dart:81:8)

Tried using the method like this:

` final File? thumbnailFile = await VideoCompress.getFileThumbnail(
        video.path,
        quality: 30, // default(100)
        position: 1, // default(-1)
      );
`

Anyone else experiencing this or has a solution?

**Comment by exmozz** [comment]
@MaxSchilling same issue with both getFileThumbnail and getMediaInfo

**Comment by exmozz** [comment]
This pull-request could be used to fix bot getFileThumbnail and getMediaInfo issue:
https://github.com/jonataslaw/VideoCompress/pull/103

**Comment by MaxSchilling** [comment]
only looks like the getMediaInfo error in the commit - asked to have the getFileThumbnail fix implemented as well

**Comment by exmozz** [comment]
@MaxSchilling I mean that fix implementation will be the same.

**Comment by MaxSchilling** [comment]
@exmozz fair point, I created https://github.com/jonataslaw/VideoCompress/pull/106 for it as fix!

**Comment by mingV2019** [comment]
use
video_compress:
    git:
      url: https://github.com/jonataslaw/VideoCompress.git

**Comment by jawa0919** [comment]
same here

**Comment by omateuscastro** [comment]
Same here!
